deployments/docker/Dockerfile

```diff
@@ -0,0 +1,136 @@
+# Args needs to be renewed for each build stage that uses these args
+ARG workDir=/app
+
+FROM golang:latest AS build-base
+
+# Renew Arg and set working directory to /app
+ARG workDir
+WORKDIR $workDir
+
+# Copy go installation files
+COPY ./go.mod ./
+COPY ./go.sum ./
+
+# Install build dependencies and tools
+RUN apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install \
+        --assume-yes \
+        --no-install-recommends \
+        apt-transport-https \
+        apt-utils \
+        jq \
+        tree \
+        protobuf-compiler \
+        python3 \
+        python3-pip \
+        libprotobuf-dev \
+        libsqlite3-dev \
+        sqlite3 \
+        zlib1g-dev \
+    && apt-get clean \
+    && apt-get autoremove --assume-yes \
+    && rm -rf /var/lib/apt/lists/* /var/tmp/* /tmp/*
+
+# Install config templating dependencies
+RUN pip install Jinja2
+
+# Download Go modules
+RUN go mod download &&\
```

Comment:
I think, we need protoc also in this list ? Kindly check

Reply:
The above list contains all the modules needed in the PSA demo and I used the [ psa demo docs](https://github.com/veraison/docs/tree/main/demo/psa/prov-verif-e2e) as a reference for what modules to include. So maybe protoc isnt needed?

Reply:
The issue with the current need to depend on protoc is documented in the tracking issue: #67